android/engine/src/main/java/org/smartregister/fhircore/engine/di/CoreModule.kt

```diff
@@ -35,7 +35,7 @@ import org.hl7.fhir.r4.utils.FHIRPathEngine
 import org.smartregister.fhircore.engine.util.helper.TransformSupportServices
 
 @InstallIn(SingletonComponent::class)
-@Module(includes = [NetworkModule::class, DispatcherModule::class, WorkManagerModule::class])
+@Module
```

Comment:
Any reasons for removing these? The other modules were included because some provider functions within CoreModule depend on dependencies from the included modules.

Reply:
From this [comment](https://github.com/google/dagger/issues/3209#issuecomment-1038398478), the other modules would already have been installed in SingletonComponent hence the `includes` is unnecessary